<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>415</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QGraphicsView" name="mainView">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>10</y>
      <width>256</width>
      <height>111</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="verticalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
    <property name="horizontalScrollBarPolicy">
     <enum>Qt::ScrollBarAlwaysOff</enum>
    </property>
   </widget>
   <widget class="QPushButton" name="playButton">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>142</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Play!</string>
    </property>
   </widget>
   <widget class="QPushButton" name="retryButton">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>140</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <property name="text">
     <string>Retry</string>
    </property>
   </widget>
   <widget class="QLabel" name="scoreLabel">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>150</y>
      <width>51</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>150</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="racerLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>210</y>
      <width>46</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="trackLabel">
    <property name="geometry">
     <rect>
      <x>140</x>
      <y>210</y>
      <width>46</width>
      <height>13</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="difficultyLabel">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>210</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QComboBox" name="comboBox">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>210</y>
      <width>69</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <item>
     <property name="text">
      <string>red</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>pink</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>green</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>orange</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>candy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>rarce</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>batman</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="trackBox">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>210</y>
      <width>71</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <item>
     <property name="text">
      <string>Day</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Night</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Candyland</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Beach</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="difficultyBox">
    <property name="geometry">
     <rect>
      <x>330</x>
      <y>210</y>
      <width>91</width>
      <height>22</height>
     </rect>
    </property>
    <property name="focusPolicy">
     <enum>Qt::NoFocus</enum>
    </property>
    <item>
     <property name="text">
      <string>Easy</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Medium</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>Hard</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>415</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
